doc_source/cni-upgrades.md

```diff
@@ -26,8 +26,12 @@ Use the following procedures to check your CNI plugin version and upgrade to the
 
 **To upgrade the Amazon VPC CNI plugin for Kubernetes**
 + If your CNI version is earlier than 1\.7\.5, and you are managing the plugin yourself, then use the appropriate command below to update your CNI version to the latest recommended version\. If your cluster is running Kubernetes `1.18` or later with `eks.3` platform version or later, and the plugin is managed by Amazon EKS, then to update the plugin, see [Configure an Amazon EKS add\-on](update-cluster.md#update-cluster-add-ons)\.
-**Important**  
-Any changes you've made to the plugin's default settings on your cluster can be overwritten with default settings when applying the new version of the manifest\. To prevent loss of your custom settings, download the manifest, change the default settings as necessary, and then apply the modified manifest to your cluster\. 
+
+**Note**  
+When applying the new manifest file, some of the custom settings might be overwritten\. For an example, environment variable AWS_VPC_K8S_CNI_CUSTOM_NETWORK_CFG is reset to false and might cause undesirable behavior leading to application downtime as Pods get assigned to default subnets\. The roadmap for CNI include items to make this process easier and seamless. Meanwhile, consider custom upgrade scenarios if you have CNI custom networking in place\. 
+
+Also, check [Configure an Amazon EKS add\-on](update-cluster.md#update-cluster-add-ons) for configuring VPC CNI. The VPC CNI add-on reduces the amount of work you need to do handle the upgraes when custom networking settings are involved.
```

Comment:
Nit: Upgrades spelling needs correction. 

Also instead of "custom networking settings", you can just mention "custom settings" since this will involve all environment variables which Cx manages.